src/libraries/TransientAllowance.sol

```diff
@@ -3,6 +3,8 @@ pragma solidity ^0.8.24;
 
 /// @notice This is a temporary library that allows us to use transient storage (tstore/tload)
 /// TODO: This library can be deleted when we have the transient keyword support in solidity.
+/// @dev The custom storage layout keyword does not work for transient storage
+/// since transient storage is automatically cleared between transactions and does not persist
```

Comment:
```suggestion
/// @dev The custom storage layout keyword does not work for transient storage.
/// However, since transient storage is automatically cleared between transactions and does not persist, custom storage is not needed.
```

Reply:
something like that?